withastro/docs
# [i18nIgnore] Integration READMEs in docs!

#### Description (required)

 This performs all the `content` changes required for the Integration READMEs :
 - Each `/en/` page has updated frontmatter:
 		- No more dev warning
 		- `githubURL` changed to `githubIntegrationURL` to separate logic for `EditLink.astro` and `PageTitle.astro` (Error messages and config-ref still need to use `githubURL` for off-site editing, but the integrations need a header link back to the package on GitHub, too.
 		- `hasREADME: true` deleted
 - the dev warning and video component imports have been removed (Only Tailwind needs the video import)
 - added a code comment to `/scripts/generate-integration-pages.ts` so that this file is included. Will need Chris to review what changes (deletion?) needs to be made here.
 - `EditLink.astro` ended up with an extra blank line, but at least it's here for easy reference in case Chris decides he prefers to tackle this from that end, vs my strategy to update `PageTitle.astro` logic.
 
 Tested behaviour:
- Dev server doesn't crash  :sweat_smile: 
- Builds fine!
- Individual integration pages work
- The integration headers still work: the github link goes to the package
- The edit this page on the integration pages works, now pointing to the current page in docs

Accompanying tasks outside of docs code:
- remove CI task
- Merge corresponding `withastro/astro` and `withastro/adapters` PRs:
		- [Cloudflare](https://github.com/withastro/adapters/pull/108)
		- [Netlify](https://github.com/withastro/adapters/pull/109)
		- [Core integrations](https://github.com/withastro/astro/pull/9479)
- Make current docs content changes corresponding to:
		- [Not yet merged Solid feature](https://github.com/withastro/astro/pull/6791/files#diff-7612e1e1749634879a227c7e1362b864b1648cf6ca4d6b06968bf995e3f1f7ca)
- Make sure no new PRs are opened to integration/adapter READMES
